classes/registration/BootServiceContainer.php

```diff
@@ -92,7 +92,7 @@ protected function registerDomPDF()
         AliasLoader::getInstance()->alias('PDF', Facade::class);
 
         $this->app->bind('dompdf.options', function () {
-            if ($defines = $this->app['config']->get('offline.mall::pdf.defines')) {
+            if ($defines = $this->app['config']->get('offline.mall::pdf.options')) {
```

Comment:
This looks like a breaking change to me. We need to change this so it uses `options` but if not defined falls back to the old `defines` property of the configuration.

Reply:
I actually reverted this, because the else statement already uses the `options`. I also changed the `barryvdh/laravel-dompdf` dependency to `^0.9|^1.0|^2.0` so it's backwards compatible.